scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -225,6 +225,24 @@ function Get-DCOSVersion {
     Throw "ERROR: Cannot find the DC/OS version from any of the masters $($masterIPs -join ', ') within a timeout of $timeout seconds"
 }
 
+function Get-MesosFlags {
+    $masterIPs = Get-MasterIPs
+    $timeout = 7200.0
+    $startTime = Get-Date
```

Comment:
7200 (12 minutes) is a very long time. Do we really expect an agent take this long to response?

Reply:
This is for the masters. We query the API from there. When deploying the hybrid cluster with 3 masters, it takes a while until the cluster is ready and responsive.

I've added 12 minutes to cover the worse case scenario. However, this is not a concern since the function will return the `Flags` as soon as the API is responsive and not wait anymore.